pprofile.py

```diff
@@ -417,6 +409,89 @@ def callgrind(self, out, filename=None, commandline=None, relative_path=False):
                     duration *= 1000000
                     print(lineno, hits, int(duration), int(duration / hits), file=out)
 
+    def get_stats(self, filename=None, commandline=None, relative_path=False):
+        """
+        Store annotated source code with current profiling statistics to python
+        dictionary object and return.
+        Time unit: second.
+        filename (str, collection of str)
+            If provided, dump stats for given source file(s) only.
+            If unordered collection, it will get sorted by decreasing total
+            file score (total time if available, then total hit count).
+            By default, list for all known files.
+        commandline (anything with __str__)
+            If provided, will be output as the command line used to generate
+            this annotation.
+        relative_path (bool)
+            For compatibility with callgrind. Ignored.
+        """
+
+        file_dict = self.file_dict
+        convertPath = _qCacheGringAdapt(_path_resolver(relative_path))
+
+        total_time = self.total_time
+        command_profile = {
+            'duration': total_time,
+            'command_line': commandline,
+            'command_profile': [],
+        }
+
+        if total_time:
```

Comment:
This test will prevent any output in statistic mode, as no time is tracked but only hits. I think it can be dropped and contained block be un-indented without any other change.